br/pkg/lightning/log/log.go

```diff
@@ -220,12 +221,13 @@ func (task *Task) End(level zapcore.Level, err error, extraFields ...zap.Field)
 	default:
 		verb = " failed"
 		extraFields = nil
+		errDetail = zap.Error(err)
 	}
 	if ce := task.WithOptions(zap.AddCallerSkip(1)).Check(level, task.name+verb); ce != nil {
 		ce.Write(append(
 			extraFields,
 			zap.Duration("takeTime", elapsed),
-			ShortError(err),
+			errDetail,
```

Comment:
i don't think this change should be contained in this pr

Reply:
rollbacked😂